packages/core-react/CHANGELOG.md

```diff
@@ -1,10 +1,19 @@
 # @livepeer/core-react
 
+## 3.2.2
+
+### Patch Changes
+
+- [#566](https://github.com/livepeer/ui-kit/pull/566) [`7a3d1f3`](https://github.com/livepeer/ui-kit/commit/7a3d1f316ae146f889758387bcfc5997dccec5c3) Thanks [@0xcadams](https://github.com/0xcadams)! - **Fix:** resolved an issue where using `disableProgressListener` would cause the player to continue to increment waittime when the player was playing.
```

Comment:
```suggestion
- [#566](https://github.com/livepeer/ui-kit/pull/566) [`7a3d1f3`](https://github.com/livepeer/ui-kit/commit/7a3d1f316ae146f889758387bcfc5997dccec5c3) Thanks [@0xcadams](https://github.com/0xcadams)! - **Fix:** resolved an issue where using `disableProgressListener` would cause the player to continue to increment waittime when the player was playing. 
```